package.json

```diff
@@ -46,10 +46,11 @@
       "eslint --fix --quiet"
     ],
     "*.{js,json}": [
-      "prettier --write"
+      "prettier --write --ignore-path=.prettierignore"
```

Comment:
Not sure if this has much effect; keeping previous behavior to avoid surprises just in case.